osu.Game.Tests/Visual/Multiplayer/TestScenePlaylistsSongSelect.cs

```diff
@@ -87,6 +87,7 @@ public override void SetUpSteps()
             {
                 Ruleset.Value = new OsuRuleset().RulesetInfo;
                 Beatmap.SetDefault();
+                SelectedMods.Value = Array.Empty<Mod>();
```

Comment:
Just to confirm, this isn't required anymore, right?

Reply:
See commit message - it's just there for safety/expected behvaiour.